r2k.c

```diff
@@ -46,6 +46,7 @@ static struct device *r2k_dev_ph;
 static struct class *r2k_class;
 static struct cdev *r2k_dev;
 static dev_t devno;
+unsigned long stack_guard_gap = STACK_GUARD_GAP;
```

Comment:
Try with `extern unsigned long stack_guard_gap;` and lets see if it works.

Otherwise remove this and rename the define to this instead. Although this might cause problems cause this value varies between kernels...

Reply:
I've actually given this a shot as well, but without success. That's  why I chose for the current approach. 

As for cross compatibility, starting from 4.11.7 its default value's has continuously been 250UL << PAGE_SHIFT

Reply:
Ok, then remove this line and rename the define to lowercase.

Reply:
Unfortunately that's not going to work. `stack_guard_gap` is declared as an `extern`, and thus (afaik) should be defined in r2k.c.